src/main/java/wolox/training/models/Book.java

```diff
@@ -70,6 +72,7 @@ public int getPages() {
     }
 
     public void setPages(int pages) {
+        Preconditions.checkNotNull(pages, "pages must be not null");
```

Comment:
Add the following validations:

- That the field is not empty

- That the field value is greater than zero